.github/workflows/publish_docs_to_wiki.yml

```diff
@@ -13,6 +13,16 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 30
     steps:
+      - name: Cancel previous runs
+        uses: styfle/cancel-workflow-action@0.8.0
```

Comment:
- Should we [use `concurrency` attributes in workflows to ensure only one job is run for each workflow](https://github.com/nimblehq/compass/pull/1228) instead?
- I'm curious whether we need to add this step to other workflows as well: both [template workflows](https://github.com/nimblehq/android-templates/tree/develop/.github/workflows) & [sample workflows](https://github.com/nimblehq/android-templates/tree/develop/.cicdtemplate).

Reply:
@ryan-conway @lydiasama To make it easy, should we `move the update to cancel the workflow to a separate task` and apply it for the rest too?

Reply:
@luongvo You may be right, I'll remove this step for now and create an issue to use concurrency on our workflows 💪🏻

Reply:
Updated in [7ba2728](https://github.com/nimblehq/android-templates/pull/456/commits/7ba27285d6bdfe75d563fb184228444848006a17)

Reply:
Created [chore ticket](https://github.com/nimblehq/android-templates/issues/462) 🙏🏻